detox/e2e/test/products/channels/localization/language.e2e.ts

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Can't find this test in the results - https://mattermost-detox-report.s3.amazonaws.com/19596752992-detox_pixel_4_xl-34-6ebc04d8de8dacce3be3480b11e2ef1195672ee1-HEAD/jest-stare/android-report.html

Along with others too to name a few:
- detox/e2e/test/products/channels/messaging/message_permalink.e2e.ts
- detox/e2e/test/products/channels/search/cross_team_search.e2e.ts
- detox/e2e/test/products/channels/search/cross_team_search.e2e.ts